packages/frontity-org-theme/src/components/scroll-button.tsx

```diff
@@ -0,0 +1,42 @@
+import { connect, styled } from "frontity";
+import { Connect } from "frontity/types";
+import React from "react";
+
+import FrontityOrg from "../../types";
+
+const ScrollButton: React.FC<Connect<FrontityOrg, any>> = ({ state }) => {
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
+  };
+
+  const Button = styled.div`
+    @media only screen and (min-width: 768px) {
+      display: none;
+    }
+    position: fixed;
+    bottom: 16px;
+    right: 16px;
+    background-color: ${state.theme.colors.frontity};
+    height: 56px;
+    width: 56px;
+    border-radius: 50%;
+    cursor: pointer;
+    text-align: center;
+    box-shadow: 0 4px 8px 0 rgba(12, 17, 43, 0.12),
+      0 1px 4px 0 rgba(12, 17, 43, 0.16);
+    color: ${state.theme.colors.white};
+    line-height: 56px;
+    font-size: 24px;
+    font-weight: bold;
+  `;
+  return typeof window !== "undefined" ? (
+    <Button onClick={scrollToTop}>
+      <p>↑</p>
```

Comment:
Is there a reason for wrapping the icon is a `<p>` tag? Doesn't seem accessibility friendly.

Reply:
Nope :smile: I deleted it.